### PR TITLE
feat: add header and footer

### PR DESCRIPTION
**Pull request by amateima**
This PR contains mostly html and css work:

- added [Header](https://github.com/UMAprotocol/website/tree/epic/new-website-layout/components/Header) and [Footer](https://github.com/UMAprotocol/website/tree/epic/new-website-layout/components/Footer) components wrapped in a [layout component](https://github.com/UMAprotocol/website/tree/epic/new-website-layout/layouts/Default)
- added base components for Buttons and Links

⚠️ Still didn't manage to get Prettier to work with `yarn lint` command. I'm kind of stuck on this

### CODE REVIEW

**Review by daywiss — APPROVED** [review]
looks good. we can probably add a task for getting prettier to work separately

File: components/Footer/components.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+`;
+
+export const NewsletterInputContainer = styled.div<{ highlighted: boolean }>`
+  display: flex;
+  border-bottom: 1px solid;
+  border-color: ${({ highlighted }) => highlighted ? "var(--primary)" : "rgba(0, 0, 0, 0.3)"};
+  transition: border-color .2s ease-out;
+
+  svg {
+    color: rgba(0, 0, 0, 0.5);
```
[the remark this answers is not in the document]
Comment (amateima) [reply components/Footer/components.tsx]: 👍

File: components/Footer/components.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import styled from "@emotion/styled";
+import { QUERIES } from "../../utils";
+import { BaseButton } from "../Button";
+import { Link } from "../Link";
+
+export const Container = styled.footer`
+  border-top: 1px solid rgba(0, 0, 0, 0.15);
```
[the remark this answers is not in the document]
Comment (amateima) [reply components/Footer/components.tsx]: 👍

File: components/Header/components.tsx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    padding: 30px 40px;
+  }
+`;
+
+export const LogoLink = styled(Link)`
+  color: var(--primary);
+  display: flex;
+
+  svg {
+    height: 20px !important;
```
[the remark this answers is not in the document]
Comment (amateima) [reply components/Header/components.tsx]: I managed to get rid of it

**Review by Tulun — APPROVED** [review]

**Pull request merged**